src/DynamoCoreWpf/Commands/SearchCommands.cs

```diff
@@ -48,5 +48,28 @@ public DelegateCommand HideSearchCommand
             }
         }
 
+        private DelegateCommand importLibrary;
+        public DelegateCommand ImportLibraryCommand
+        {
+            get
+            {
+                if (importLibrary == null)
+                    importLibrary = new DelegateCommand(this.dynamoViewModel.ImportLibrary,
+                        this.dynamoViewModel.CanImportLibrary);
+                return importLibrary;
+            }
+        }
+
+        private DelegateCommand showPackageManagerSearch;
+        public DelegateCommand ShowPackageManagerSearchCommand
+        {
+            get
+            {
+                if (showPackageManagerSearch == null)
+                    showPackageManagerSearch = new DelegateCommand(this.dynamoViewModel.ShowPackageManagerSearch,
+                        this.dynamoViewModel.CanShowPackageManagerSearch);
+                return showPackageManagerSearch;
+            }
+        }
```

Comment:
It does look like we need these commands in `DynamoViewModel` instead of being here, right? Here's what I recommend us do:

1. Add `ImportLibraryCommand` and `ShowPackageManagerSearchCommand` commands in `DynamoViewModel` class.
2. Make `ImportLibrary`, `CanImportLibrary`, `ShowPackageManagerSearch`, `CanShowPackageManagerSearch` methods private in `DynamoViewModel`.
3. Move these command creation logic to `DynamoViewModel`.
4. Update this file to:

```csharp
public DelegateCommand ImportLibraryCommand
{
    get { return dynamoViewModel.ImportLibraryCommand; }
}
```

That's the best way to avoid changing a lot of things in this PR.